src/utils/CMakeLists.txt

```diff
@@ -2,15 +2,17 @@
 # Utility sources
 # =============================================================================
 set(UTIL_SOURCE_FILES
-    ${CMAKE_CURRENT_SOURCE_DIR}/common_utils.hpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/string_utils.hpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/common_utils.cpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/perf_stat.hpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/perf_stat.cpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/table_data.hpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/table_data.cpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/logger.hpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/logger.cpp
+    common_utils.cpp
```

Comment:
why did you change this?

Reply:
I needed to add `common_utils.[ch]pp` and used that opportunity to
* sort the list to improve maintainability
* remove `${CMAKE_CURRENT_SOURCE_DIR}` which is implicit for `add_library` CMake function.